### DISCUSSION TITLE
What are the correct settings to make use of the new autofill service

### DISCUSSION THREAD

**Discussion by fourstepper** [zeapo/Android-Password-Store#1364] [Q&A] [answered]
Hi,

I am trying to get autofill work with Android 10 and Bromite. I am not sure how to set it up or if it just doesn't work for me right now, before with other apps I have just used the autofill accessibility service with display over other apps.

Thank you!

**Comment by msfjarvis** [comment, marked as answer]
You just need to go into the app's settings and turn on Autofill, the prompts that follow are pretty detailed.

**Reply by fourstepper** [reply]
Right, that's what I did as well - the Autofill service is set to password store, but the autofill dialogue doesn't come up when I go to a site for which I have a login saved

![nextcloud_login](https://user-images.githubusercontent.com/49439044/113267338-8790c300-92d6-11eb-8aaf-32976a4027ff.png)

The scheme for this password is custom_directory/<domain>/username, but I have tried a bunch of others with a different scheme and it doesn't work either

**Reply by fmeum** [reply]
Every phone comes with an allowlist for browsers supported by the Autofill compatibility mode. You could try adding Bromite to that list via adb as follows:

```
$ adb shell settings put global autofill_compat_mode_allowed_packages "$(adb shell settings get global autofill_compat_mode_allowed_packages):org.bromite.bromite[url_bar]"
```

After restarting Bromite, Autofill prompts should start showing.

**Reply by fourstepper** [reply]
This is great. Thank you, it worked.
